mkdocs.yml

```diff
@@ -204,6 +204,7 @@ plugins:
   - search:
       separator: '[\s\-,:!?=\[\]()<>{}"/\\]+|\.(?!\d)|&[lg]t;'
   - awesome-pages
+  - privacy
```

Comment:
I tried the server preview in Codespaces. I used the Network tab of the browser to see the GET requests, for example:

| Environment | Font source |
| --- | --- |
| Current production | `fonts.gstatic.com` |
| With privacy plugin | Preview server domain |

Looks like the privacy plugin actually works. 🥳 

I scrolled and clicked around, and I did not see anything obviously broken. 😄 The build step passes without errors.